engine/src/main/scala/cromwell/engine/workflow/ValidateActor.scala

```diff
@@ -78,7 +65,7 @@ class ValidateActor(wdlSource: WdlSource, workflowInputs: Option[WdlJson], workf
     * 2.) Raw inputs and it's coercion
     * 3.) Workflow Options
     * 4.) RuntimeAttributes w.r.t. a backend
-    * @return
+    * @return The result of the validation as instance of `ValidationResult`
     */
   private def validateAll(): Future[ValidationResult] = {
```

Comment:
The only standard `Future` that never fails is `Future.successful`.  The `receive` method of an actor returns `Unit`, so this really is all about the side effects.  The use of `map` is misleading if it makes the situation appear otherwise.  IMHO having the `recover` here rather than buried in the method that returns `Future[ValidationResult]` makes this all more clear.